src/link_manager/LinkManagerStateMachine.cpp

```diff
@@ -856,11 +858,9 @@ void LinkManagerStateMachine::handleDefaultRouteStateNotification(const std::str
 
     if (mComponentInitState.test(MuxStateComponent)) {
         if (ms(mCompositeState) != mux_state::MuxState::Label::Standby && routeState == "na") {
-            CompositeState nextState = mCompositeState;
-            enterLinkProberState(nextState, link_prober::LinkProberState::Wait);
-            switchMuxState(nextState, mux_state::MuxState::Label::Standby, true);
-            LOGWARNING_MUX_STATE_TRANSITION(mMuxPortConfig.getPortName(), mCompositeState, nextState);
-            mCompositeState = nextState;
+            mSendPeerSwitchCommandFnPtr();
```

Comment:
I don't fully understand this chunk of diff. Here we don't know if xcvrd is healthy or not. The change assumed that xcvrd is not healthy?

Reply:
Even if xcvrd is healthy, it leaves no harm to do the switch this way. Send peer switch command includes the chunk. 

mMuxPortPtr->setMuxState() is to handle the case when mux is in wait state: the chunk will be skipped in this case. When it's not in wait state, it's no harm the set the same value twice.